Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "armor"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
Could we leave the version out of this PR? I prefer to version things as part of a release (which would also need to be _minor_, not _patch_ in this case).

Reply:
Yup, my bad!